iOS internship.xcodeproj/project.pbxproj

```diff
@@ -10,12 +10,15 @@
 		415BBC37215A5565009FA99C /* Freddy.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 41DB2DD3215A5551003F9E7A /* Freddy.framework */; };
 		415BBC39215B59C5009FA99C /* CityModel.swift in Sources */ = {isa = PBXBuildFile; fileRef = 415BBC38215B59C5009FA99C /* CityModel.swift */; };
 		415BBC3B215B5A4F009FA99C /* CityTransport.swift in Sources */ = {isa = PBXBuildFile; fileRef = 415BBC3A215B5A4F009FA99C /* CityTransport.swift */; };
+		415BBC41215B804A009FA99C /* ForecastViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 415BBC40215B804A009FA99C /* ForecastViewController.swift */; };
+		415BBC43215B89DA009FA99C /* ForecastModel.swift in Sources */ = {isa = PBXBuildFile; fileRef = 415BBC42215B89DA009FA99C /* ForecastModel.swift */; };
+		415BBC45215B89EC009FA99C /* ForecastTransport.swift in Sources */ = {isa = PBXBuildFile; fileRef = 415BBC44215B89EC009FA99C /* ForecastTransport.swift */; };
+		415BBC47215BAA1E009FA99C /* Protocols.swift in Sources */ = {isa = PBXBuildFile; fileRef = 415BBC46215BAA1E009FA99C /* Protocols.swift */; };
```

Comment:
This shouldn't exist. All protocols should be owned by object files, if it's related to model, it should be defined in related model file.